app/uk/gov/hmrc/apiplatform/modules/submissions/repositories/QuestionnaireDAO.scala

```diff
@@ -416,14 +416,23 @@ object QuestionnaireDAO {
 
       val question3 = MultiChoiceQuestion(
         QuestionId("57d706ad-c0b8-462b-a4f8-90e7aa58e57a"),
-        Wording("Where are your servers that store customer information?"),
+        Wording("Where are your servers that process customer information?"),
         Statement(
+          StatementText("For cloud software, check the server location with your cloud provider."),
+          CompoundFragment(
+            StatementText("Learn about "),
+            StatementLink("adequacy agreements (opens in a new tab)", "https://ico.org.uk/for-organisations/dp-at-the-end-of-the-transition-period/data-protection-and-the-eu-in-detail/adequacy/"),
+            StatementText(" or "),
+            StatementLink("check if a country has an adequacy agreement (opens in a new tab)", "https://ico.org.uk/for-organisations/dp-at-the-end-of-the-transition-period/data-protection-and-the-eu-in-detail/the-uk-gdpr/international-data-transfers/#:~:text=Andorra%2C%20Argentina%2C%20Canada%20(commercial,a%20finding%20of%20adequacy%20about"),
```

Comment:
this link matches the one in the prototype but not the one in the ticket, not sure which is correct

Reply:
The ticket is out of date - the screenshot still has the accordion.  I have assumed that the prototype is correct.